### PR TITLE
WIP: Exporting the workspace / all workspaces as HAR file

### LINKED ISSUE #416: [Feature Request] Export request/response history as HAR
It would be great if Insomnia could export the request/response history into a [.har](https://en.wikipedia.org/wiki/.har) file.

This would allow integrating Insomnia with other HTTP-related tools (my use case is [HTTPolice](https://github.com/vfaronov/httpolice)).

I can actually see something called `exportHar` in the Insomnia codebase, but I can’t figure out how to invoke it, or whether it’s what I’m looking for at all. In any case, the [Import/Export](https://insomnia.rest/documentation/import-export/) documentation page only mentions *importing* from HAR.

HAR is a poorly defined format, but it’s the best we’ve got, and by navigating certain pitfalls it should be possible to achieve decent interoperability.

### PR DESCRIPTION
**Pull request by Atvaark**
This PR adds a the option to export the requests in a workspace (or all workspaces) as an HAR (HTTP Archive 1.2) file.

See: #416 
---
Exporting as .har is implemented.
Exporting single har requests still works as it looks (the code generator).
I'm currently testing the output in different tools (e. g. Fiddler or HTTPolice) to check for any issues.

Closes #416

### CODE REVIEW

**Comment by gschier** [comment]
Thanks for this @Atvaark! I'll try and get to reviewing/merging this tomorrow.

File: app/models/response.js [hunk 1]
```diff
@@ -89,13 +89,15 @@ export function remove (response: Response) {
...
 
-export function findRecentForRequest (requestId: string, limit: number) {
-  return db.findMostRecentlyModified(type, {parentId: requestId}, limit);
+export async function findRecentForRequest (requestId: string, limit: number): Promise<Array<Response>> {
+  const responses = await db.findMostRecentlyModified(type, {parentId: requestId}, limit);
+  return responses;
 }
 
-export async function getLatestForRequest (requestId: string) {
+export async function getLatestForRequest (requestId: string): Response | null {
```
Comment (gschier) [inline app/models/response.js]: Minor, but this should return `Promise<Response | null>`.

**Comment by Atvaark** [comment]
Hey @gschier,
I added a couple of tests. Please take a look at them in case I missed something.

**Comment by gschier** [comment]
Thanks, I will try to take a look today.

**Comment by gschier** [comment]
Sorry for the delay @Atvaark. I was without a computer for the last two weeks because my keyboard broke. I have this on my list of to-dos for next week, though. 😄

**Comment by gschier** [comment]
[Merge Pull 523](https://trello.com/c/j7IsQnlR/635-merge-pull-523)

**Pull request merged**